srw_detector.py

```diff
@@ -127,6 +128,9 @@ def get_options():
             watchpoints.append(data['models']['beamline'][i]['title'] +
             ', position: ' + str(data['models']['beamline'][i]['position']))
     print(f'WATCHPOINTS:       {watchpoints}')
+    if len(watchpoints) < 1:
+        print("No watchpoint found. This simulation will not work.")
+        sys.exit()
```

Comment:
`sys.exit()` is not recommended, it may brake things when it's used as a library. Better use `raise ValueError('<your text>')` in this case (or `raise Exception('<...>')` as a more general exception).

Reply:
This has been fixed.